system/services/assetManager/AssetManagerService.cfc

```diff
@@ -936,6 +936,26 @@ component {
 		return settings;
 	}
 
+	public any function listEditorDerivates(){
+		var derivatives = _getConfiguredDerivatives();
+		var publicDerivatives = ["none"];
```

Comment:
The "none" doesn't belong here really. It is specific logic for the form control and not the service logic. If I do `.listEditorDerivatives()` I expect to just get actual valid derivatives.